conf/parallel.conf.js

```diff
@@ -3,17 +3,39 @@ var config = {
     'browserstack.user': process.env.BROWSERSTACK_USERNAME || 'BROWSERSTACK_USERNAME',
     'browserstack.key': process.env.BROWSERSTACK_ACCESS_KEY || 'BROWSERSTACK_ACCESS_KEY',
     'build': 'mocha-browserstack',
-    'name': 'parallel_test',
     'browserstack.debug': 'true',
   },
   'multiCapabilities': [{
-      'browserName': 'Chrome'
-    },{
-      'browserName': 'Safari'
-    },{
-      'browserName': 'Firefox'
-    },{
-      'browserName': 'IE'
+      'os_version': '10',
+      'browserName': 'Chrome',
+      'browser_version': 'latest',
+      'os': 'Windows',
```

Comment:
minor nitpick here and for other caps.. keep os and os_version keys next to each other.